src/domain/relation/getFollowers.ts

```diff
@@ -7,8 +7,9 @@ import createView from './view/createView';
 
 export default async function getFollowers(requester: Requester, followingId: string): Promise<RelationView[]>
 {
-    const data = await retrieveByFollowing(followingId);
-    const relationData = await Promise.all(data.map(data => retrieve(requester.id, data.followerId)));
+    const followingData = await retrieveByFollowing(followingId);
 
-    return Promise.all(relationData.map(createView));
+    const requesterData = await Promise.all(followingData.map(data => retrieve(requester.id, data.followerId)));
```

Comment:
Why have you changed this to requesterData? It's an array of relationData.

Reply:
Both arrays are relation data. The first contains the relations from the following (viewed profile) perspective, the second contains the relations from the requester perspective (viewer). So I tried to make this situation more clear.

Reply:
Clear. Let's keep it this way and see if we need to change this in the future.